09-schema/00-overview.md

```diff
@@ -5,29 +5,85 @@ permalink: /docs/schema/overview
 
-An **entity** can have attributes and play roles in relations.
+If you know a bit about other schema first database knowledge systems, you know that normally database design involves three schemas:
 
-An **attribute** expresses a *value* of a specified *datatype*. It can have attributes of its own and play roles in relations.
+  1. _A high-level conceptual schema_, that models your problem and usually involves some variation of the entity-relation model
 
-A **relation** has at least one roleplayer, can have attributes and play roles in other relations.
+  1. _A mid-level logical schema_, that depends on the database type you are using (for example if you are going relational, this would involve turning the conceptual model into tables and going over a series of normalisation steps of your schema)
 
-In the sections that follow, by looking at various real-world examples, we learn how these concepts can be defined in a schema to represent a dataset.
+  1. _A low level physical schema_, that requires you to optimise your schema according to how your physical resources are distributed
+
+With Grakn, thanks to our high level knowledge model, your schema will closely resemble the conceptual schema, essentially avoiding the hassle of going through the other two modelling steps. The Grakn system will take care of those.
+
+This greatly simplifies the design process, getting you what can be considered a highly normalised distributed schema without the need of going through the logical and physical modelling.
+
+Let us have a look at the main components of a Grakn knowledge graph and schema.
+
+## Grakn Data Model
+
+### Concepts
+Everything that describes our domain in a Grakn Knowledge Graph is a concept. This includes the elements of the schema (namely types and roles, which we call schema concepts) and the actual data instances. 
+We refer to data instances as things - they can be thought of as instances of types defined in the schema.
+
+### Types
+Types constitute the core of your schema. They provide the necessary vocabulary to talk about our domain. They come in three flavours: [Entities](/docs/schema/concepts#entity), [Relationships](/docs/schema/concepts#relationship), and [Attributes](/docs/schema/concepts#attribute):
+
+__Entities__ are the main actors in our domain. These are usually the type of things we want to know about. Entity types provide means of classifying the objects in our domain.
 
-Lastly, we have the [**Graql Rules**](/docs/schema/rules). Rules watch for schema-driven patterns in the data and infer queryable conclusions. Rules are one way to perform automated reasoning in a Grakn knowledge graph.
+__Relationships__ are things that connect other concepts. Each relationship can connect a number of things, as specified in your schema. The character of participation in a relationship is characterised by roles that can be played in that relationship. Each relationship is required to
```

Comment:
To be exact relationships connect things, so maybe this to reduce overuse of _thing_:
___Relationships__ connect other things together. Each relationship can connect a number of things_

Reply:
Previously using character twice in this sentence:
_The participation of a thing in a relationship is characterised by a **role** that it plays in that relationship._ - Role should be emphasised